packages/cluster-ui/src/queryFilter/filter.module.scss

```diff
@@ -56,7 +56,7 @@ $dropdown-hover-color: darken($colors--background, 2.5%);
       }
 
       &__margin-top {
-        margin-top: 27px;
+        margin-top: 15px;
```

Comment:
Not flagging necessarily, but do we use any tokenization for gutter values?

Reply:
We use tokens for sizes of text, button, things like that. When it's a very specific space for a specific component I use the value itself, because it's very unlikely this value will be aligned with any other components